src/data/external/osmosis.ts

```diff
@@ -64,13 +70,12 @@ export const useGammTokens = () => {
   const gammTokens = new Map<string, string>()
 
   if (fetch.data) {
-    for (const [poolId, poolAsset] of Object.entries(fetch.data) ?? {}) {
+    for (const [poolId, poolAsset] of Object.entries(fetch.data)) {
       gammTokens.set(
         "gamm/pool/" + poolId,
         poolAsset.map((asset) => asset.symbol).join("-") + " LP"
```

Comment:
The only issue I can think of is if one of the `asset`s in `poolAsset` return without the `symbol`.  Should we add a protection for this edge case?

Reply:
good call.  we can't assume anything here unfortunately.  will update to add a conditional ? on that symbol.